packages/macro/test/jsx-trans.ts

```diff
@@ -414,35 +414,37 @@ const cases: TestCase[] = [
       `,
   },
   {
-    name: "Keep forced newlines",
+    name: "Strip whitespace around tags but keep whitespaces in JSX containers",
```

Comment:
by "JSX container" you mean this? `{"Wonderful framework "}` - just clarifying what that term means in this context